elpis/engines/kaldi/inference/gmm-decode-conf/gmm-decode-conf.sh

```diff
@@ -103,7 +103,8 @@ utils/int2sym.pl -f 5 \
 
 # Now, wav.scp needs to be in segment form
 # eg audio_id filename
-echo "decode audio.wav" > ./data/infer/split1/1/wav.scp
+audio_filename=$(<./data/infer/audio_meta.txt)
+echo "decode ${audio_filename}" > ./data/infer/split1/1/wav.scp
```

Comment:
Read the filename in from a text file, seems like a convenient way to pass data between Python and shell